orienteer-core/pom.xml

```diff
@@ -10,119 +10,264 @@
 	<artifactId>orienteer-core</artifactId>
 	<packaging>jar</packaging>
 	<name>orienteer-core</name>
+	<properties>
```

Comment:
I recommend do not create properties for versions if that property has been used only once.